### PR TITLE
Feature/#1 create task

### LINKED ISSUE #1: User can create a task
A user can create a task.

A Task consists of: 
* task name
* description
* Due Date (optional)
* Estimated time (optional)
* Reminder (optional)

acceptance criteria:
* Create button exists
* description box
* possibility to add due date, estimated time and reminder

### PR DESCRIPTION
**Pull request by ghost**
closes #1

### CODE REVIEW

**Review by theexiile1305 — CHANGES_REQUESTED** [review]
- The view `app/src/main/res/layout/fragment_second.xml` should be changed as following:
  - Change the title of the title field from _Due Date_ to _Title_ 
  - All fields should be requiered.
  - In the estimated time fields only integers should be there.
- Please uses tests as well alongside the implementation
- Remove all comments from the file `edu.hm.cs.ma.todoguru.SecondFragment`
- Please use the new Date/Time API instead of the deprecated one (Calendar, Date, etc.)

File: app/src/main/java/edu/hm/cs/ma/todoguru/database/Task.kt [hunk 1]
```diff
@@ -1,10 +1,21 @@
...
 data class Task(
     @PrimaryKey(autoGenerate = true)
-    var id: Long = 0L
+    var id: Long = 0L,
+    @ColumnInfo(name = "title")
+    var title: String,
+    @ColumnInfo(name = "description")
+    var description: String,
+    @ColumnInfo(name = "due Date")
+    var dueDate: String,
```
[the remark this answers is not in the document]
Comment (akfxhll) [reply app/src/main/java/edu/hm/cs/ma/todoguru/database/Task.kt]: Is done

File: app/build.gradle [hunk 2]
```diff
@@ -63,6 +63,9 @@ dependencies {
     implementation 'androidx.constraintlayout:constraintlayout:1.1.3'
     implementation 'androidx.navigation:navigation-fragment-ktx:2.2.2'
     implementation 'androidx.navigation:navigation-ui-ktx:2.2.2'
+    // material
+    implementation 'com.google.android.material:material:1.0.0-beta01'
```
[the remark this answers is not in the document]
Comment (akfxhll) [reply app/build.gradle]: Not really, I removed it

**Comment by theexiile1305** [comment]
Well, done 😄

**Review by theexiile1305 — APPROVED** [review]

**Pull request merged**